src/components/Menu/MenuGridItem/MenuGridItem.jsx

```diff
@@ -73,6 +74,7 @@ const MenuGridItem = forwardRef(
         ref={mergedRef}
         className={cx("menu-grid-item--wrapper", className)}
```

Comment:
no css module class?

Reply:
Stylesheet contained only this:
`@import "../../../styles/themes";`
So, I've removed it, but left the original classnames to keep existing overrides